tests/mysql_test.js

```diff
@@ -0,0 +1,28 @@
+import sql from 'k6/x/sql';
+
+// The second argument is a MySQL connection string, e.g.
+// myuser:mypass@tcp(127.0.0.1:3306)/mydb
+const db = sql.open('mysql', '');
+
+export function setup() {
+  db.exec(`
+    CREATE TABLE IF NOT EXISTS keyvalues (
+      id INT(6) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+      \`key\` VARCHAR(50) NOT NULL,
+      value VARCHAR(50) NULL
+    );
+  `);
+}
+
+export function teardown() {
+  db.close();
+}
+
+export default function () {
+  db.exec("INSERT INTO keyvalues (`key`, value) VALUES('plugin-name', 'k6-plugin-sql');");
+
+  let results = sql.query(db, "SELECT * FROM keyvalues WHERE `key` = ?;", 'plugin-name');
+  for (const row of results) {
+    console.log(`key: ${String.fromCharCode(...row.key.toString().split(','))}, value: ${String.fromCharCode(...row.value.toString().split(','))}`);
```

Comment:
Hhmm why is this `split()` and conversion required?

Reply:
Key and Name from results display as ASCII code. Then I convert it to String.

ASCII
<img width="1021" alt="CleanShot 2564-09-28 at 09 19 32@2x" src="https://user-images.githubusercontent.com/2748846/135012133-2f3d6649-7b39-4641-a06e-84d22a526463.png">


After convert
<img width="1037" alt="CleanShot 2564-09-28 at 09 18 57@2x" src="https://user-images.githubusercontent.com/2748846/135012147-fa214f91-0bf3-4465-8f0b-e558990cda08.png">

Reply:
I understand, but I'm wondering why this is required for MySQL and not for the other RDBMSs.

We can merge this as is, but I'd like to clarify this, and I just don't have the time right now.